kumascript/macros/LiveSampleURL.ejs

```diff
@@ -4,25 +4,27 @@
 //
 // Parameters:
 //
-// $0   The ID of the sample's section or containing block.
-// $1   The URL from which to pull the sample; if not provided,
+// $0 - The ID of the sample's section or containing block.
+// $1 - (Deprecated)  The URL from which to pull the sample; if not provided,
 //      the current page is assumed.
 
-var pagePath = env.url;
+let pagePath = env.url;
+let baseUrl = env.live_samples.base_url;
+let samplePath = `${pagePath}/runner.html`.replace(/\/\/+/g, "/");
 
 if ($1 && $1.length > 0) {
+    mdn.deprecatedParams("The second parameter of 'LiveSampleURL' is deprecated");
```

Comment:
Ideally, the message would have an actionable suggestion on how to proceed. I guess in this case the original example has to be copied into the page, right?